install/Cargo.toml

```diff
@@ -44,3 +43,6 @@ winreg = { workspace = true }
 
 [package.metadata.docs.rs]
 targets = ["x86_64-unknown-linux-gnu"]
+
+[package.metadata.cargo-udeps.ignore]
+normal = ["serde_yaml_08"]
```

Comment:
Not sure if this is how it should be, but the added line seems to be telling `cargo-udeps` to ignore `serde_yaml_08`, but, at the same time, import of `serde_yaml_08` is removed in the same PR.
Or is this just WIP?

Reply:
yep. as this pr title says... ;)